src/eval.jl

```diff
@@ -117,21 +124,17 @@ handle("evalall") do data
           result = nothing
           try
             result = include_string(mod, code, path)
-          catch e
-            bt = catch_backtrace()
-            st = cliptrace(stacktrace(bt))
-            ee = EvalError(e, st)
-            if isREPL()
-              printstyled(stderr, "ERROR: "; bold=true, color=Base.error_color())
-              Base.showerror(IOContext(stderr, :limit => true), e, st)
-              println(stderr)
-            else
-              render(Console(), ee)
-            end
+          catch err
+            ee = EvalError(err, st)
+
+            # show error in REPL:
+            Base.showerror(IOContext(stderr, :limit => true), ee)
+            # show notification (if enabled in Atom):
```

Comment:
This is also the previous behaviour, but why we always don't show errors inline with `evalall` ?

Reply:
`evalall` shouldn't show anything inline, imho -- it's basically a shorthand for `include(currentFile)`.

Reply:
Okay, understand.
Other than that everything looks good to me.